### ISSUE TITLE
Crash on Windows 10 

### ISSUE THREAD

**Issue by ScriptVortexDE** [adlerluiz/ytmdesktop#1383]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

I get this error when i play music, 
The error is very sporadic.

It comes when it comes, can be after 5 minutes or only after 2 hours 

### Expected Behavior

YouTube Music Desktop App Crashed

Environment Details:
    YouTube Music Desktop App 2.0.5
    Electron 29.0.1
    Windows NT 10.0.19045

Name: Error
Message: getaddrinfo ENOTFOUND lh3.googleusercontent.com
Cause: Unknown

Error: getaddrinfo ENOTFOUND lh3.googleusercontent.com
    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:118:26)

### Steps To Reproduce

Reproduce in itself kp, 

Just listen to music and at some point it will come, it could be just with me or with others I don't know

### YTMDesktop

2.05

### OS

Windows

### OS Version

10

### Arch

x64

### Installation way

.exe

**Comment by Alipoodle** [comment]
Duplicate of #1383
